pom.xml

```
@@ -17,14 +17,14 @@
   <parent>
     <groupId>com.nablarch</groupId>
     <artifactId>nablarch-parent</artifactId>
-    <version>5-NEXT-SNAPSHOT</version>
+    <version>5u24</version>
```

ここも同様。5u25(5-NEXT-SNAPSHOT)であるべきですが
外部貢献者は5-NEXT-SNAPSHOTにアクセスできません。
ビルド時に支障があるため最後に取り込みます
```suggestion
    <version>5-NEXT-SNAPSHOT</version>
```

---

（https://github.com/nablarch/nablarch-micrometer-adaptor/pull/21#discussion_r1620159040 を参照）

---

コード上は対応しました
(余談ですが、nablarch-parentについては5-NEXT-SNAPSHOTの取り込みができませんでした。)
https://central.sonatype.com/artifact/com.nablarch/nablarch-parent/5u24/overview